apps/web-client/src/lib/db/cr-sqlite/core/worker-db.worker.ts

```diff
@@ -0,0 +1,111 @@
+import * as Comlink from "comlink";
+
+import type { DBAsync, _TXAsync, OnUpdateCallback, TXCallback, VFSWhitelist } from "./types";
+
+import { getCrsqliteDB } from "./init";
+
+export type MsgInitOk = { _type: "wkr-init"; status: "ok" };
+export type MsgWkrError = { _type: "wkr-init"; status: "error"; error: string; stack?: string };
+export type MsgInit = MsgInitOk | MsgWkrError;
+
+async function start() {
+	try {
+		const [dbname, vfs] = self.name.split("---") as [string, VFSWhitelist];
```

Comment:
Could be helpful to note that this is constructed in line 18 of `./worker-db.ts`. Took me a sec to figure this out